api/routes/party.js

```diff
@@ -9,6 +9,7 @@ const Party = require('../models/Party');
 // @desc    Adds a new party to the database
 // @access  Private
 router.post('/add', (req, res) => {
+  // ? Does this comment need to change to refer to Parties instead?
```

Comment:
No, the party has a list of tables (this used to be tableNumbers), which should be an array of the Table ObjectIds. Good catch though